app/routes/groups_.new.tsx

```diff
@@ -0,0 +1,67 @@
+import { Card } from '~/components/ui/containers';
+
+export default function GroupNew() {
+  return (
+    <div className="grow py-20 lg:py-40 bg-secondary w-full flex flex-col items-center justify-center">
+      <div className="flex w-full items-center justify-center flex-col">
+        <div className="max-w-[900px] w-full flex flex-col gap-3">
+          <h1 className="text-black font-bold text-4xl">Create New Group</h1>
+          <h2 className="text-black font-bold text-2xl">Your Community Starts Here</h2>
+          <Card>
+            <div className="flex pt-4">
+              <div className="flex-col">
+                <div className="w-1/2 pb-4">
```

Comment:
Haven't touched this form yet. Just took it over from the existing code. We will iterate on the form in the next step (follow-up PR).